lib/transform.mli

```diff
@@ -1,5 +1,6 @@
-val translate : float -> float -> Shape.shape -> Shape.shape
-val scale : float -> Shape.shape -> Shape.shape
-val rotate : int -> Shape.shape -> Shape.shape
-val compose : ('a -> 'b) -> ('b -> 'c) -> 'a -> 'c
-val repeat : int -> (Shape.shape -> Shape.shape) -> Shape.shape -> Shape.shape
+type transform = (Shape.shape -> Shape.shape)
```

Comment:
I like `transformation` better for the name of a type. `transform` would have been appropriate for the name of a function.